mesonbuild/linkers/linkers.py

```diff
@@ -978,7 +978,7 @@ def __init__(self, for_machine: mesonlib.MachineChoice,
                          version=version)
 
     def get_link_whole_for(self, args: T.List[str]) -> T.List[str]:
-        if not args:
+        if not args or len(args) == 1:
```

Comment:
Assuming the type annotation is correct and `args` cannot be `None`, wouldn't this be nicer if spelt
```python
if len(args) < 2:
    return args
return self._apply_prefix('--start-group') + args + self._apply_prefix('--end-group')
```
or something like this?

Reply:
Yes.